seqr/utils/elasticsearch/es_search.py

```diff
@@ -248,7 +250,7 @@ def _filter_by_genotype(self, inheritance_mode, inheritance_filter, quality_filt
                         continue
 
             if not genotypes_q:
-                for family_guid in sorted(family_samples_by_id.keys()):
+                for family_guid in family_samples_by_id.keys():
```

Comment:
I found `sorted()` is unnecessary here because the order is not important for the codes in the loop.

Reply:
After removing the sorting on the filters, the `sorted()` here becomes necessary. Added it back.